Products/ZenUI3/browser/resources/js/zenoss/monitoringTemplates/threshold.js

```diff
@@ -364,4 +408,56 @@
         }
     });
 
+    Ext.define('RPNValues', {
+        extend: 'Ext.data.Model',
+        fields: [
+            {name: 'name', mapping: 'name'},
+            {name: 'maxrpn', mapping: 'maxrpn'},
+            {name: 'minrpn', mapping: 'minrpn'},
+            {name: 'rpnvalue', mapping: 'rpnvalue'}
```

Comment:
Since the `name` and `mapping` configs have the same values, the `mapping` configs can be removed.  You should be able to specify the `fields` config very concisely, e.g.
```
    fields: ['name', 'maxrpn', 'minrpn', 'rpnvalue'],
```